OnlineTeamScan_V2_BE/Tests/Tests/TeamMemberServiceTests.cs

```diff
@@ -0,0 +1,216 @@
+using AutoMapper;
+using BL.Services.TeamMemberServices;
+using Common.DTOs.TeamMemberDTO;
+using DAL.Models;
+using DAL.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using System;
+using Xunit;
+
+namespace Tests.Tests
+{
+    public class TeamMemberServiceTests
+    {
+        private readonly TeamMemberService _service;
+        private readonly Mock<IUnitOfWork> _unitOfWork = new Mock<IUnitOfWork>();
+        private readonly IMapper _mapper = MapperConfig.Initialize();
+
+        private int id = 1;
+        private string email = "john.smith@gmail.com";
+        private string firstname = "John";
+        private string lastname = "Smith";
+        private bool isActive = true;
+        private int teamId = 1;
+
+        public TeamMemberServiceTests()
+        {
+            _service = new TeamMemberService(_unitOfWork.Object, _mapper);
+        }
+
+        [Fact]
+        public void GetTeamMemberById_ShouldReturnTeamMember_WhenTeamMemberExists()
+        {
+            // Arrange
```

Comment:
Goed dat je dit gebruikt in opbouw. Dat doe ik ook. Ik vraag me echter af of dit thuishoort in productie code, nadat het dev-proces is voltooid?